src/ClientImpl.cpp

```diff
@@ -378,7 +378,8 @@ void ClientImpl::initiateAuthentication(PendingConnection* pc, struct buffereven
         throw voltdb::LibEventException();
     }
     AuthenticationRequest authRequest( m_username, "database", m_passwordHash, m_hashScheme );
-    ScopedByteBuffer bb(authRequest.getSerializedSize());
+    int size = authRequest.getSerializedSize();
```

Comment:
what I mean is original code ScopedByteBuffer bb(authRequest.getSerializedSize()); does not require size var at all and you can leave it as is.